source/includes/_create_product_home3.md

```diff
@@ -0,0 +1,67 @@
+## `/products/create_product/home3`
+
+> A POST request to this endpoint will create a new home 3 product on the system using an existing pcb (jackal pcb)
+
+```python
+    url = "https://smartapi.bboxx.co.uk/v1/products/create_product/home3"
+    data = json.dumps({
+        'pcb_imei': '110932356179789',
+        'battery': {
+            'part_type_id': 18004, 
+            'serial_number': 'madeupserialnumb3r'
+            }
+            })
```

Comment:
Fix indentation.

Reply:
done

Reply:
Others have been fixed, but this one still needs to be fixed.